mail/src/main/java/org/jboss/as/mail/extension/DirectMailSessionService.java

```diff
@@ -40,7 +40,7 @@ public class DirectMailSessionService extends MailSessionService {
     private final SessionProvider provider;
 
     public DirectMailSessionService(final SessionProvider provider) {
-        super(null);
+        super(null, null);
```

Comment:
This is probably out of scope for this, but this being a subclass is clunky. AFAICT this class and MailSessionService are actually completely distinct impls of Service<Session>.

@yersan fyi